requirements.txt

```diff
@@ -2,4 +2,4 @@ pyyaml >= 5.1
 pydantic
 numpy >= 1.17
 git+https://github.com/lsst/pex_config@main#egg=lsst-pex-config
-git+https://github.com/lsst/pipe_base@main#egg=lsst-pipe-base
```

Comment:
I don't understand, these two lines are exactly the same. Does one of them end in a newline? Github usually points that out, I'm just trying to understand what the diff is.

Reply:
Yes, the new one ends in a newline (note also that it's 163 bytes instead of 162). I don't know why the git diff makes it so hard to interpret.

Reply:
Ok. Github used to highlight that there was a new line in a diff, it's a shame that they took that feature away.